cogs/welcome/welcome.py

```diff
@@ -315,4 +504,5 @@ async def setImage(self, ctx: Context, imageUrl: str = None):
     async def test(self, ctx: Context):
         """Test the welcome DM by sending a DM to you."""
         await self.sendWelcomeMessage(ctx.message.author, test=True)
+        # await self.sendWelcomeMessageChannel(ctx.message.author, ctx.guild)
```

Comment:
if we're not using this, can we please remove it?

Reply:
done, could add it back if we can get some sort of test channel that it can be passed as wel